packages/ses/src/permits.js

```diff
@@ -1233,14 +1237,90 @@ export const permitted = {
 
   // *** Control Abstraction Objects
 
+  // https://github.com/tc39/proposal-iterator-helpers
+  Iterator: {
+    // Properties of the Iterator Constructor
+    '[[Proto]]': '%FunctionPrototype%',
+    prototype: '%IteratorPrototype%',
+    from: fn,
+  },
+
   '%IteratorPrototype%': {
     // The %IteratorPrototype% Object
     '@@iterator': fn,
+    // https://github.com/tc39/proposal-iterator-helpers
+    constructor: 'Iterator',
+    map: fn,
+    filter: fn,
+    take: fn,
+    drop: fn,
+    flatMap: fn,
+    reduce: fn,
+    toArray: fn,
+    forEach: fn,
+    some: fn,
+    every: fn,
+    find: fn,
+    '@@toStringTag': 'string',
+    // https://github.com/tc39/proposal-async-iterator-helpers
+    toAsync: fn,
+  },
+
+  // https://github.com/tc39/proposal-iterator-helpers
+  '%WrapForValidIteratorPrototype%': {
+    '[[Proto]]': '%IteratorPrototype%',
+    next: fn,
+    return: fn,
+  },
```

Comment:
Is it odd that this one doesn't have an `@@toStringTag` but the next one does? Likewise for the async cases below.

Reply:
I think the wrap iterator is still one expected to not be revealed? I'm not sure, going by memory here.

Reply:
I believe this is indeed and oversight, and posted about it on the I believe erroneously closed issue: https://github.com/tc39/proposal-iterator-helpers/issues/42#issuecomment-1622659282

Reply:
Thanks! Resolving here.